css/index.css

```diff
@@ -46,29 +50,45 @@ nav ul li {
   color: salmon;
 }
 
-h2 {
-  color: gray;
+h2, label {
+  color: #1565c0;
 }
 
+#about-summary  {
+  margin-left: 10px;
+}
+
+/* allows the jobtitle and company name to stay on one 1 line at w375px */
+ul {
+  padding: 10px;
+}
+
+.experience-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-row-gap: 10px; 
+
+}
+
+/* 
```

Comment:
I wouldn't keep commented-out code around without some explanation... When should it be uncommented?